xml/System.IO/File.xml

```diff
@@ -3335,13 +3335,13 @@
       </Parameters>
       <Docs>
         <param name="path">The file to open for reading.</param>
-        <summary>Opens a text file, reads all lines of the file, and then closes the file.</summary>
-        <returns>A string containing all lines of the file.</returns>
+        <summary>Opens a text file, reads all text of the file, and then closes the file.</summary>
+        <returns>A string containing all text of the file.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method opens a file, reads each line of the file, and then adds each line as an element of a string. It then closes the file. A line is defined as a sequence of characters followed by a carriage return ('\r'), a line feed ('\n'), or a carriage return immediately followed by a line feed. The resulting string does not contain the terminating carriage return and/or line feed.  
+ This method opens a file, reads the whole text of the file, and returns it as a string. It then closes the file.
```

Comment:
nit: "reads the while file" or "reads all the text in the file" are better than "reads the whole text"